.ci/integration.cloudbuild.yaml

```diff
@@ -445,8 +445,25 @@ steps:
           "Firestore" \
           firestore \
           firestore
-    
 
+  - id: "alloydbwaitforoperation"
+    name: golang:1
+    waitFor: ["compile-test-binary"]
+    entrypoint: /bin/bash
+    env:
+      - "GOPATH=/gopath"
+    secretEnv: ["CLIENT_ID"]
+    volumes:
+      - name: "go"
+        path: "/gopath"
+    args:
+      - -c
+      - |
+        .ci/test_with_coverage.sh \
+          "Alloydb Wait for Operation" \
+          alloydbwaitforoperation \
```

Comment:
```suggestion
          utility \
```

Reply:
@Yuan325 can you confirm this is the right set up (https://github.com/googleapis/genai-toolbox/blob/main/.ci/test_with_coverage.sh)